Makefile

```diff
@@ -57,7 +57,7 @@ E2E_COMMIT_ID?=local
 E2E_KUBECONFIG?=""
 
 # Kubernetes version to use in envtest for controller tests.
-ENVTEST_K8S_VERSION ?= 1.30.x
```

Comment:
Out of curiosity since it seems unrelated, why did we make this change?

Reply:
I just realized we're using an old version of Kubernetes control plane and just updated while writing the docs. This is not used anywhere except dev environments. In the CI, we're using our full matrix to test against all Kubernetes version we support.